src/components/hooks/index.js

```diff
@@ -0,0 +1,3 @@
+import useTheme from "./useTheme";
```

Comment:
Can we move this to the root `src` level just to keep consistency with the other files in the project? We don't have many files yet — I'll take care of re-organizing the repo if needed in the future 👍